docker-compose.ove.yml

```diff
@@ -1,17 +1,17 @@
 version: '3'
 services:
-  ove-ove-latest:
-    image: ovehub/ove:latest
+  ove-ove-0-2-0:
+    image: ovehub/ove:0.2.0
     ports:
     - "8080:8080"
 
-  ove-external-tuoris-latest:
-    image: ovehub/ove-external-tuoris:latest
+  ove-external-tuoris-0-2-0:
```

Comment:
@oserban, please set Tuoris version to latest until Tuoris 1.0.0 (or whichever version) is released.